.drone.yml

```diff
@@ -1205,58 +1086,6 @@ depends_on:
 - default-linux-amd64
 - default-windows-1809
 
----
-kind: pipeline
-name: docker-image-digests-windows-2004
-
-platform:
-  os: windows
-  arch: amd64
-  version: 2004
-
-# remove this and use upstream images when https://github.com/drone/drone-git/pull/25 is merged
-clone:
-  disable: true
-
-steps:
-- name: clone
-  image: luthermonson/drone-git:windows-2004-amd64
```

Comment:
the faster we get this guy out of the code base the better.

Reply:
20h2 removal will be coming soon enough